### ISSUE TITLE
Add warning on the GoogleApps homepage for TestDocumentList

### ISSUE THREAD

**Issue by oanat** [xwikisas/application-googleapps#7]
We should add a warning on the GoogleApps homepage for the link `http://<server>/xwiki/bin/view/GoogleApps/TestDocumentList` since it's only available for Google accounts (XWiki accounts get a 400 connection error "Error: redirect_uri_mismatch").

**Comment by polx** [comment]
Yes, this should be done by an extension of the GoogleApps service which should have a method `hasUserGAppsProfile` or so.
